app/assets/javascripts/initialize.js

```diff
@@ -1,4 +1,5 @@
-$(document).ready(function() {
+
+init = function() {
```

Comment:
I think this needs to be `var init` otherwise, `init` becomes a global variable.